bower.json

```diff
@@ -8,12 +8,10 @@
     "!src/**/*"
   ],
   "dependencies": {
-    "ringcentral": "^3.1.0",
```

Comment:
Do we still need bower?

Reply:
I think we could remove bower support. https://github.com/ringcentral/ringcentral-web-phone/issues/352

Reply:
Done